CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Added
 
 - \[[#304](https://github.com/rust-vmm/vm-memory/pull/304)\] Implement ReadVolatile and WriteVolatile for TcpStream
+- \[[#311](https://github.com/rust-vmm/vm-memory/pull/311)\] Allow compiling without the ReadVolatile and WriteVolatile implementations
```

Comment:
This needs to move up into the "Upcoming version" section. Rest lgtm!

Reply:
Oops, sorry about that.